src/docker-compose.yml

```diff
@@ -25,6 +25,8 @@ services:
       - DJANGO_SECRET_KEY=really-long-random-string-BNPecI7+s8jMahQcGHZ3XQ5yUfRrSibdapVLIz0UemdktVPofDKcoy
       # Run Django in debug mode on local
       - DJANGO_DEBUG=True
+      # Set DJANGO_LOG_LEVEL in env
+      - DJANGO_LOG_LEVEL
```

Comment:
**Question:** Would we need to set it to "DEBUG" just for the development side?

Reply:
No, the default, if not supplied, is DEBUG, on local.  In the manifests for each sandbox, this is set to INFO.

Reply:
TIL, thanks Dave!